docs/en/ingest-management/faq.asciidoc

```diff
@@ -227,3 +226,29 @@ and port for your setup. If you run everything locally, the address is
 you can copy the {es} endpoint URL from the overview page of your deployment.
 If you're not running in {ecloud}, make sure the {kib} and {es} HTTPS ports
 are both accessible; by default these are `5601` and `9200` respectively.
+
+[discrete]
+[[how-do-i-reset-a-missing-dashboard-asset]]
+== If I delete an integration dashboard asset from {kib}, how do I get it back?
+
+To reinstall the assets for a specific integration, you can use the UI for most cases
+and the {fleet} API for those not allowed in the UI.
+In order to use the API, you'll need to know the package name and version
+you want to reinstall.
```

Comment:
```suggestion
To reinstall the assets for a specific integration, you can use the {fleet} API using the package name and version.
```
we don't offer a "reinstall" UI, the closest is uninstall + install, but we don't allow uninstall when the package is in use in a policy so this probably won't work for most cases. what do you think of removing the part about UI and just reference the API?

Reply:
ugh, you're right.